guides/releasing/Release Workflow.md

```diff
@@ -495,86 +443,164 @@ Once everything above is completed and Apple has approved the iOS client, the fi
-| [6.3. Deploy new docs](#63-deploy-new-docs)                                                     |
-| [6.4. Add related packages to versions endpoint](#64-add-related-packages-to-versions-endpoint) |
+## 5.4. Re-publish project templates
+
+**Why:** Ensure that the templates include the latest version of packages, so when we release the beta
+
+**How:** Follow [0.9. Publish `next` project templates](#09-publish-next-project-templates) but be sure that the published template has the `sdk-xx` tag on npm in addition to `next`.
+
+## 5.5. Promote versions to production with new SDK version flagged as beta
+
+| Prerequisites                                 |
+| --------------------------------------------- |
+| **All previous steps** |
 
 **Why:** It's time for everything that uses the production versions endpoint to know about this new SDK version!
 
 **How:**
 
+- `et update-versions-endpoint -s ${SDK_MAJOR_VERSION}.0.0 -k 'beta' -v 'true'`
 - `et promote-versions-to-prod`
 - Double check every change before pressing `y`!
 
-## 6.6. Promote packages to `latest` on NPM registry
+## 5.6. Add SDK support to Snack
+
+| Prerequisites                                                 |
+| ------------------------------------------------------------- |
+| [2.6. Publish any missing or changed packages](#26-publish-any-missing-or-changed-packages) |
+| [4.2. Making a simulator/emulator build](#42-making-a-simulatoremulator-build) |
+
+**How:** Reach out to Hein (@ijzerenhein)
+
+## 5.7. Announce beta availability
+
+**Why:** We want interested developers to try it out and report any issues they encounter.
+
+**How:**
+
+Publish a blog post that includes the following information:
+- Link to a GitHub umbrella issue for beta release issues
+- Link to CHANGELOG
+- Provide instructions for how to opt-in
+
+## 5.8. Test, fix, and monitor
 
-**Why:** Previously we've published packages and now, if everything works like a charm, we can promote those packages to `latest` on NPM.
+**Why:** The beta period will run for approximately 1 week. During that time we should try to discover regressions and new bugs, fix them, and roll the fixes out to the beta users. The fixes will require repeating many of the previous steps, such as re-submitting an iOS build and re-deploying Turtle.
 
 **How:**
 
-- Use the `et promote-packages` script (TODO: add more detailed instructions once this script lands)
+- Monitor GitHub issues.
+- Expo team should all update any dogfooding apps they work on, including re-building standalone apps.
+- Test out new features.
+- Report updates in the umbrealla issue.
+- Fix, test, repeat.
+
+# Stage 6 - Final release
+
+**If today is Friday:** You may want to submit the iOS app for approval, but otherwise STOP! Wait until next week to finish the release :) 
+
+## 6.1. Release iOS/Android clients to the general public
+
+**How:**
 
-## 6.7. Publishing final project templates
+- **iOS**:
```

Comment:
All of this submission stuff needs to be done earlier on, ideally ~1-3 days before we actually plan on releasing, and once we're reasonably sure the beta build has no major bugs. Should probably be part of section 5. Text for 6.1 iOS should be just "log into App Store Connect and press 'Release This Version' on the approved release."